labware-library/src/labware-creator/fieldsToLabware.ts

```diff
@@ -65,7 +65,9 @@ export function fieldsToLabware(
     // Also note that 'irregular' in `format` just means "not 96/384 standard, not trough, and not trash",
     // it doesn't imply anything about having multiple grids or not.
     const format = 'irregular'
-    let quirks: string[] = []
+    const lpcQuirk =
+      fields.hasLpcQuirk === 'true' ? 'noLabwarePositionCheck' : null
```

Comment:
```suggestion
      fields.hasLpcQuirk ? 'noLabwarePositionCheck' : null
```
Nit, in JS, we generally drop the strict equality check for booleans and either test the truthiness by doing the above or use the bang operator to test falsyness.

Reply:
I need to check strict equality because hasLpcQuirk is coming in as a string